### PR TITLE
feat: added landing page

### PR DESCRIPTION
**Pull request by varnita619**
- added landing page 

Screenshot (75)

### CODE REVIEW

File: src/Pages/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./HomePage.css";
+const HomePage = () => {
+  return (
+    <div className=" main-container flex justify-center items-center">
+      <div className="home-img">
+        <img src="./img/home-img.png" alt="" className="original-img" />
```
Comment (shrey27) [inline src/Pages/HomePage/HomePage.jsx]: don't leave alt attribute empty

File: src/Pages/HomePage/HomePage.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  position: absolute;
+  top: 3.5rem;
+  left: 29rem;
+  width: 15.8rem;
+  height: 34rem;
+  animation-name: fade;
+  animation-timing-function: ease-in-out;
+  animation-iteration-count: infinite;
+  animation-duration: 4s;
+  animation-direction: alternate;
```
Comment (anuj-kumary) [inline src/Pages/HomePage/HomePage.css]: You can use **animation** shorthand property here

**Review by shrey27 — APPROVED** [review]
code is well structured
PR is good to merge

**Review by anuj-kumary — APPROVED** [review]
Code looks clean.
You can merge it.

File: src/Pages/HomePage/HomePage.css (hunk 2 above)
Comment (varnita619) [reply src/Pages/HomePage/HomePage.css]: Done

File: src/Pages/HomePage/HomePage.jsx (hunk 1 above)
Comment (varnita619) [reply src/Pages/HomePage/HomePage.jsx]: Done

**Pull request merged**